doc/source/rest.j2

```diff
@@ -119,24 +162,25 @@ timestamp:
 
 {{ scenarios['get-measures-from']['doc'] }}
 
+Aggregate
```

Comment:
Maybe `Aggregation` is better. not to be confused with 'aggregates'. and the parameter in url uses `aggregation` as well